### PR TITLE
Move Building instruction's to a BUILDING.md and add new deps for fedora and arch

### PR DESCRIPTION
**Pull request by RyzenDew**
Adds new instruction's for cloning the git and grabbing 3rd party modules

### CODE REVIEW

**Comment by RyzenDew** [comment]
About to add arch deps and fedora deps

**Comment by RyzenDew** [comment]
This is ready for review i did a major clean up

File: .github/readme.md [hunk 1]
```diff
@@ -23,47 +23,93 @@ If you want to contribute as a developer, please contact us in the [Discord](htt
...
 
-`mkdir ps4-fs`
+```
+mkdir ps4-fs
+```
 
-``sudo mount -t exfat -o uid=`id -u`,gid=`id -g` ./ps4-hdd.exfat ./ps4-fs``
+```
+sudo mount -t exfat -o uid=`id -u`,gid=`id -g` ./ps4-hdd.exfat ./ps4-fs
+```
```
Comment (eg-ayoub) [inline .github/readme.md:88]: is it possible to have all of these in a single block or in a one-liner?
it looks very chunky when rendered

Comment (RyzenDew) [reply .github/readme.md]: great idea

**Review by eg-ayoub — APPROVED** [review]

File: .github/readme.md [hunk 2]
```diff
@@ -23,47 +23,95 @@ If you want to contribute as a developer, please contact us in the [Discord](htt
...
+> Arch will have to use xbyak from the aur for now
 
 ## Cloning the Repo
 
-``git clone https://github.com/RPCSX/rpcsx && cd rpcsx``
+```
+git clone --recursive https://github.com/RPCSX/rpcsx && cd rpcsx
+```
+
+> if you get a cmake error run
```
Comment (elad335) [inline .github/readme.md:67]: Does it really need the > here? I viewed it on your branch abd it looks weird imo.

**Comment by IsaacMarovitz** [comment]
Might be time to make a BUILDING.md so the readme isn't too cluttered

File: .github/readme.md (hunk 2 above)
Comment (RyzenDew) [reply .github/readme.md:67]: nope not anymore the issue was solved

**Comment by RyzenDew** [comment]
> Might be time to make a BUILDING.md so the readme isn't too cluttered

all done

**Comment by elad335** [comment]
Why two BUILDIND.md?

**Comment by RyzenDew** [comment]
> Why two BUILDIND.md?

cause github hates me and thank you for telling me i fixed it

**Pull request merged**